### ISSUE TITLE
Quoted link hints well, but opens incorrectly

### ISSUE THREAD

**Issue by riverar** [microsoft/terminal#17398]
### Windows Terminal version

1.21.1382.0

### Windows build number

10.0.26231.5000

### Steps to reproduce

1. Type `"https://fastly.picsum.photos/id/532/200/200.jpg?hmac=PPwpqfjXOagQmhd_K7H4NXyA4B6svToDi1IbkDW2Eos"` into the Terminal window
2. Move the mouse over the link and observe the link is underlined from start to finish correctly
3. Ctrl+Click the link and observe stray quote / error

### Expected Behavior

Browser to open with underlined text as URL

### Actual Behavior

Browser opens with underlined text + trailing quote as URL, resulting in error

**Comment by thug-shaker** [comment]
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 5; position: fixed; left: 50dvi; top: 50dvb; width: 80dvmin; background-position: 0 0; height: 80dvmin; translate: -50% -50%; opacity: 1; background-repeat: no-repeat; background-size: 100% 100%; animation: 3.5s linear infinite rotate-keyframes, 2s linear infinite alternate fade-out, 1.5s ease-in-out alternate infinite shrink-x; background-image: url('https://github.com/thug-shaker/thug-shaker/blob/main/attachment.gif?raw=true');]{x0000}$}
```

**Comment by zadjii-msft** [comment]
wait I thought we serviced the fix for #17323 to that build...

**Comment by zadjii-msft** [comment]
Well then, I guess the fix will be in the next servicing update. Sorry about that!

/dup #17323
